Documentation/minimum_requirements.md

```diff
@@ -0,0 +1,14 @@
+#.NET Core Minimum System Requirements:
+
+In general, the minimum requirements of your operating system will be enough to run .NET Core applications.
+
+##1.X 
+ 
+The minimum available resources required for a simple standalone application are:
+ 
+- 25MB Disk Space
+- 16MB available RAM
+
+Depending on your application and performance requirements, these requirements may be higher. 
+
+*SSE2 processor instruction support is required.
```

Comment:
This seems lost in the text. Perhaps move it to the first paragraph? Also, should we link somewhere or give more info on what this means? How would a customer know if their computer has that? It's not something we typically call out in a minimum system requirements, I think.